### ISSUE TITLE
Syntax error in pymilvus

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#6937]
#### Steps/Code to reproduce:

```
W
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 57, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 800, in load_collection
    self.wait_for_loading_collection(collection_name, timeout)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 60, in handler
    if e.code == Status.ILLEGAL_COLLECTION_NAME:
AttributeError: 'AttributeError' object has no attribute 'code'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "main.py", line 82, in run_suite
    runner.prepare(**cases[0])
  File "/src/milvus_benchmark/runners/accuracy.py", line 250, in prepare
    self.milvus.load_collection(timeout=600)
  File "/src/milvus_benchmark/client.py", line 46, in wrapper
    result = func(*args, **kwargs)
  File "/src/milvus_benchmark/client.py", line 440, in load_collection
    return self._milvus.load_collection(collection_name, timeout=timeout)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 63, in handler
    raise e
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 51, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 40, in inner
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 334, in load_collection
    return handler.load_collection("", collection_name=collection_name, timeout=timeout)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 60, in handler
    if e.code == Status.ILLEGAL_COLLECTION_NAME:
AttributeError: 'AttributeError' object has no attribute 'code'
 (milvus_benchmark.main:114)
```
#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:

**Comment by XuanYang-cn** [comment]
This seemed already to be fixed on https://github.com/milvus-io/pymilvus/pull/626

It's pymilvus-2.0.0rc3.dev15, Please check your version and see if this's a new bug or a fixed bug. @del-zhenwu

**Comment by yanliang567** [comment]
verify fix on 2.0.0rc3.dev15
